arPullGlob.py

```diff
@@ -369,7 +343,7 @@ def cavDatCnt(caVs, sT, fiN):
             bozo["CommFlt"] = clwn.count(8)
             bozo["SSAFlt"] = clwn.count(16)
             bozo["Quench"] = clwn.count(32)
-            bozo["Clips"] = len(resultsFB[pvFB]["values"])
+            bozo["Clips"] = len(resultsFB[pvFB]['values']) # was ["values"]
```

Comment:
Oh, bozo... Eventually when we figure out what this is, we want to rename it hahaha